peer/x/roundrobin/peerring.go

```diff
@@ -101,6 +101,15 @@ func (pr *PeerRing) RemoveAll() []peer.Peer {
 	return peers
 }
 
+// All returns a snapshot of all the peers from the ring as a list.
+func (pr *PeerRing) All() []peer.Peer {
```

Comment:
mmm I followed the convention of this collection. It has Add, Remove, RemoveAll, so All sounds reasonable. Else It would likely be AddPeer, RemovePeer, RemovePeers and Peers.